meta-koi/recipes-core/initrdscripts/initramfs-scripts-android/init.machine.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+BOOT_DIR=$1
+
+mkdir $BOOT_DIR/nvdata
+mount -t ext4 /dev/mmcblk0p9 $BOOT_DIR/nvdata
+
+mkdir $BOOT_DIR/firmware
+mount -t ext4 /dev/mmcblk0p32 $BOOT_DIR/firmware
+
+mkdir -p /nvdata
+mkdir -p /firmware
+mkdir -p /system/etc/wifi
+mkdir -p /system/vendor/firmware/
+mkdir -p /vendor/firmware
+
+mount --bind $BOOT_DIR/system/etc/wifi          /system/etc/wifi
+mount --bind $BOOT_DIR/system/vendor/firmware/  /system/vendor/firmware/
+mount --bind $BOOT_DIR/vendor/firmware          /vendor/firmware
+mount --bind $BOOT_DIR/nvdata                   /nvdata
+mount --bind $BOOT_DIR/firmware                 /firmware
+
+dd if=/nvdata/MACWLAN of=/proc/wifi/mac
+
+ip link set up dev wlan0
+sleep 1
+ip link set down dev wlan0
```

Comment:
Is this needed for the MACWLAN overwrite to be effective ?
If this is, could you add a comment describing why it's there ?

Reply:
I'm not sure. I believe most of this is left over from whatever watch it's based on (ray?). I haven't looked into testing/cleaning these up yet because there were bigger issues, such as the perplexing `LD_BIND_NOW` requirement up until now.

Reply:
Okay, it's not the end of the world either, it's just unfortunate to wait for 1 second in every boot if we don't even need to :)